### PR TITLE
fixed :

### PR DESCRIPTION
**Pull request by isentropic**
Thanks for writing this small library, here is a minor fix. If you do not mind, please give me write access, as I might add more stuff, as I happen to need this for my project.

### CODE REVIEW

**Comment by chaaland** [comment]
Sorry the code is so sloppy! Was making some changes and then got very busy these past couple months but you've reminded me to revisit this. Did you still want permissions? I see you forked it but in the event you still want to collaborate I can try setting that up. Just let me know

File: src/utils/ellipse_formatter.jl [hunk 1]
```diff
@@ -123,7 +123,7 @@ function parametric2quad(semiaxis_lengths; center=[0 0], ccw_angle=0)
     return S, vec(center)
 end
 
-function quad2parametric(S; center=[0 0])
```
Comment (chaaland) [inline src/utils/ellipse_formatter.jl]: Isn't semicolon necessary for keyword arguments of the form `keyword=value`

File: src/utils/ellipse_formatter.jl [hunk 2]
```diff
@@ -123,7 +123,7 @@ function parametric2quad(semiaxis_lengths; center=[0 0], ccw_angle=0)
...
+function quad2parametric(S, center=[0 0])
```
Comment (chaaland) [inline src/utils/ellipse_formatter.jl]: Isn't a semicolon necessary to specify keyword args?

File: src/EllipseFit.jl [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
-           vandermonde, least_squares_fit, orthogonal_distance_fit, gauss_newton, levenberg_marquardt
+           vandermonde, least_squares_fit, orthogonal_distance_fit, gauss_newton, levenberg_marquardt, conic2parametric
 
     include("least_squares_fit.jl")
     include("orthogonal_distance_fit.jl")
 
     ####################################
     # Utils
     ####################################
+# include("utils/elementwise_pseudoinvert.jl")
```
Comment (chaaland) [inline src/EllipseFit.jl]: remove?

File: src/utils/parametric2ellipse_coords.jl [hunk 4]
```diff
@@ -1,7 +1,5 @@
-include("utils.jl");
 
-
-function parametric2ellipse_coords(semiaxis_lengths::Vector{T}; center=[0 0], ccw_angle=0, numpoints=1000) where T <: Real
+function parametric2ellipse_coords(semiaxis_lengths::Vector{T}, center=[0; 0], ccw_angle=0, numpoints=1000) where T <: Real
```
Comment (chaaland) [inline src/utils/parametric2ellipse_coords.jl]: Semicolon to separate keyword args

File: src/utils/ellipse_formatter.jl (hunk 2 above)
Comment (isentropic) [reply src/utils/ellipse_formatter.jl]: Yeah, my mistake, I was thinking that this was causing the trouble

**Comment by isentropic** [comment]
@chaaland Yes, I'd like to collaborate. Had to fork it in order to fix critical issues

**Review by chaaland — APPROVED** [review]

**Pull request closed without merging**